app/models/edition/publishing.rb

```diff
@@ -42,7 +42,7 @@ def change_note_required?
     elsif draft? && new_record?
       false
     else
-      other_editions.published.any?
```

Comment:
This *should* be the same effect, what are you going for with this change?

Reply:
I've read in the past that `SELECT EXISTS` can be optimised by the DB more so than `SELECT 1 WHERE`, but it really is a minor/insignificant change at this point. I just think it reads better.